lib/components/Button.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+class MyButton extends StatefulWidget {
+  final String text;
+  final VoidCallback onPress;
+  int keys;
+
+  MyButton(
+      {Key key,
+      this.text,
+      this.keys,
+      this.onPress})
+      : super(key: key);
+  
+  @override
+  _MyButtonState createState() => new _MyButtonState();
+}
+
+class _MyButtonState extends State<MyButton>{
+  String _displayText;
```

Comment:
Why do you want another _displayText variable if all you are doing is accessing widget.text?